.github/workflows/nightly-android.yaml

```
@@ -180,4 +180,4 @@ jobs:
           name: changelog
           path: fastlane/metadata/android/en-US/changelogs/
       - name: Deploy to Google Play beta
-        run: bundle exec fastlane android upload_to_playstore build_number:${{ github.run_number }}
+        run: bundle exec fastlane android upload_to_playstore build_number:${{ github.run_number + 366 }}
```

actionlint (1.7.12)

[error] got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)